qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -71,7 +71,7 @@ def __repr__(self) -> str:
         queue_info = ["{}={}".format(attr, repr(value))
                       for attr, value in self.__dict__.items()]
 
-        return "{}({})".format(self.__class__.__name__, ', '.join(queue_info))
+        return "<{}({})>".format(self.__class__.__name__, ', '.join(queue_info))
```

Comment:
Can you actually format the datetime object to a more readable format? Like `'2020-01-17T23:29:47.684000+00:00'` (i.e. `.isoformat()`). If you just do a `str()` it looks like `datetime.datetime(2020, 1, 17, 23, 29, 47, 684000, tzinfo=datetime.timezone.utc)`, which is no different from the default representation. Same goes to status - it should be the value of the enum, which is more readable.